package.json

```diff
@@ -28,32 +28,32 @@
     "ethereum-blockies-base64": "^1.0.2",
     "focus-trap-react": "^10.0.0",
     "framer-motion": "^6.5.1",
-    "gatsby": "^4.21.1",
-    "gatsby-plugin-emotion": "^7.19.0",
-    "gatsby-plugin-gatsby-cloud": "^4.20.0",
-    "gatsby-plugin-image": "^2.21.0",
-    "gatsby-plugin-manifest": "^4.21.0",
-    "gatsby-plugin-matomo": "^0.9.0",
+    "gatsby": "^5.4.0",
+    "gatsby-plugin-emotion": "^8.4.0",
+    "gatsby-plugin-gatsby-cloud": "^5.4.0",
+    "gatsby-plugin-image": "^3.4.0",
+    "gatsby-plugin-manifest": "^5.4.0",
+    "gatsby-plugin-matomo": "^0.14.0",
     "gatsby-plugin-mdx": "^3.0.0",
```

Comment:
Not upgrading the mdx plugin in this PR since the amount of work for doing that is considerable.

I've created #9160 to achieve that.

Reply:
Not upgrading this plugin generates an error but is not impacting our project as far as I tested it.